build.gradle

```diff
@@ -224,6 +224,39 @@ apply from: "gradle/testSetup.gradle"
 apply from: "gradle/fix-GRADLE-2492.gradle"
 apply from: 'gradle/customM2Check.gradle'
 
+ext.publicApiIncludes = [
```

Comment:
I moved the definition of the public api to `gradle/publicApi.gradle`. We can still iterate on that in the future.